docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
+| **Throttle minimum**                    | The **minimum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
+| **Throttle maximum**                    | The **maximum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
+| **Throttle hover**                      | The throttle value the craft will use to hover at the return altitude. **It is important to set this value accurately.**                                                                                                                                                                                                                                                                                                                          |
+| **Minimum distance to home (meters)**   | The minimum distance that the craft has to be from the home position to activate GPS Rescue. If the rescue starts too close to home, the craft will fly away until this distance is reached, then start the normal rescue behavior.                                                                                                                                                                                                               |
+| **Minimum satellites**                  | The minimum number of satellites required to activate GPS Rescue. If the number of satellites is below this value, GPS Rescue will not be activated.                                                                                                                                                                                                                                                                                              |
+| **Allow arming without fix**            | Allows arming without a GPS fix. If enabled and armed before GPS fix is acquired, the craft will disarm on loss of signal. If enabled, you cannot take off without a GPS fix.                                                                                                                                                                                                                                                                     |
```

Comment:
If this option is enabled, the quad may be armed without a GPS Home location being determined.  If there is no GPS Home location, and a GPS Rescue is initiated, the quad will immediately disarm and crash.  The (much safer) default is for this option to be off, so that if GPS Rescue is set for Failsafe, arming is not possible until a Home location has been acquired.